packages/permission/package.json

```diff
@@ -20,8 +20,8 @@
   },
   "homepage": "https://github.com/orochi-network/zkDatabase#readme",
   "type": "module",
-  "main": "./build/src/index.js",
-  "types": "./build/src/index.d.ts",
+  "main": "build/src/index.js",
```

Comment:
using ./ in the second variant is more explicit and consistent, especially if other file references in your project follow this convention.

Reply:
It avoids any ambiguity about whether the path is relative or not.

Reply:
Fixed